controlplane/modules/route/internal/rib/routes.go

```diff
@@ -0,0 +1,99 @@
+package rib
+
+import (
+	"net/netip"
+	"slices"
+	"time"
+)
+
+type RouteSourceID uint8
+
+const (
+	RouteSourceUnknown RouteSourceID = iota
+	RouteSourceStatic
+	RouteSourceBird
+)
+
+type LargeCommunity struct {
+	GA    uint32
+	Data1 uint32
+	Data2 uint32
+}
+
+type LargeCommunityList struct {
+	LargeCommunity
+	Next *LargeCommunityList
+}
+
+type Route struct {
```

Comment:
This is an important structure. Please add a description for each field, what it means and where it is important.